### PR TITLE
chore(url launcher): Fix for url launcher in android OS 11

### LINKED ISSUE #84: URL links and send mail features are not working in android 11
URL links and send mail features are not working in android 11

### PR DESCRIPTION
**Pull request by akash933**
Package visibility filtering in android OS 11 is blocking app from launching URL, send mail. To fix these issues added intent for below actions:
> 1. View
> 2. Dial
> 3. Send

BUG : #84

### CODE REVIEW

File: android/app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -6,6 +6,20 @@
     <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE" />
     <uses-permission android:name="android.permission.VIBRATE"/>
     <uses-permission android:name="android.permission.RECEIVE_BOOT_COMPLETED"/>
+    <queries>
+        <intent>
+            <action android:name="android.intent.action.VIEW" />
+            <data android:scheme="https" />
```
Comment (pranavsb) [inline android/app/src/main/AndroidManifest.xml:12]: should we add another `<intent` for `http`? Just thinking if HTTP links added by users for their websites or personal links will work.

**Comment by pranavsb** [comment]
Flutter Test CI is faililng for some reason. Don't think it is related to this PR. I will check.

File: android/app/src/main/AndroidManifest.xml (hunk 1 above)
Comment (akash933) [reply android/app/src/main/AndroidManifest.xml:12]: http links are also working with current changes

**Comment by pranavsb** [comment]
Closes #84

**Comment by pranavsb** [comment]
LGTM

**Pull request merged**